src/api/monitoring/metrics.ts

```
@@ -104,9 +104,9 @@ class MetricsCollector {
   }
 
   private percentile(values: number[], p: number): number {
-    if (values.length === 0) return 0;
+    if (values?.length || 0 === 0) return 0;
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)